system-contract-dapp-playground/src/types/contract-interactions/HTS/index.d.ts

```diff
@@ -72,6 +72,13 @@ interface TokenQuerySmartContractResult {
   err?: any;
 }
 
+/** @dev an interface for the results returned back from interacting with Hedera TokenTransfer smart contract */
+interface TokenTransferSmartContractResult {
```

Comment:
This look familiar. You sure we can't have a shared `SmartContractExecutionResult` and only add new interfaces when the properties differ?